src/app/beer_garden/db/mongo/pruner.py

```diff
@@ -5,16 +5,18 @@
 
 from brewtils.stoppable_thread import StoppableThread
 from mongoengine import Q
+from mongoengine.queryset.visitor import Q as Q2
```

Comment:
The `Q` that you're importing as `Q2` here is the same `Q` as the line above.  So there's no need to import it twice.  I don't think you need Q at all though, which I will explain below.